[2025-03-17T17:47:20.289+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-02T22:00:00+00:00 [queued]>
[2025-03-17T17:47:20.321+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gen_data_daily.get_date scheduled__2025-03-02T22:00:00+00:00 [queued]>
[2025-03-17T17:47:20.330+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2025-03-17T17:47:20.371+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_date> on 2025-03-02 22:00:00+00:00
[2025-03-17T17:47:20.387+0000] {standard_task_runner.py:60} INFO - Started process 500 to run task
[2025-03-17T17:47:20.392+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gen_data_daily', 'get_date', 'scheduled__2025-03-02T22:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/gen_data_daily.py', '--cfg-path', '/tmp/tmp119bnpvd']
[2025-03-17T17:47:20.393+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask get_date
[2025-03-17T17:47:20.524+0000] {task_command.py:423} INFO - Running <TaskInstance: gen_data_daily.get_date scheduled__2025-03-02T22:00:00+00:00 [running]> on host d107d9d70093
[2025-03-17T17:47:20.813+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='longnv' AIRFLOW_CTX_DAG_ID='gen_data_daily' AIRFLOW_CTX_TASK_ID='get_date' AIRFLOW_CTX_EXECUTION_DATE='2025-03-02T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-02T22:00:00+00:00'
[2025-03-17T17:47:20.817+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:315: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-03-17T17:47:20.818+0000] {python.py:201} INFO - Done. Returned value was: 2025-03-03
[2025-03-17T17:47:20.898+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gen_data_daily, task_id=get_date, execution_date=20250302T220000, start_date=20250317T174720, end_date=20250317T174720
[2025-03-17T17:47:20.977+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-17T17:47:21.163+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
